xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -481,10 +494,18 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="P:System.Threading.CancellationToken.None" />.</param>
+        <summary>Asynchronously clears all buffers for this Brotli stream, causes any buffered data to be written to the underlying device, and monitors cancellation requests.</summary>
```

Comment:
Similarly, this description would cause some confusion rather than clarity (at least for me). Buffers are getting cleared? Where? What device are we writing to?

Look at documentation for other streams. They aren't copy-pasted from the abstract stream class and contain context specific details that match the behavior of the specific stream implementation.

Like https://docs.microsoft.com/en-us/dotnet/api/system.io.memorystream.flushasync?view=netstandard-2.1 or https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.flushasync?view=netstandard-2.1#remarks